keylime/failure.py

```diff
@@ -92,18 +95,18 @@ class Failure:
     events: List[Event]
     recoverable: bool
     highest_severity: Optional[SeverityLabel]
-    _component: Optional[Component]
+    _component: Component
     _sub_components: Optional[List[str]]
 
-    def __init__(self, component, sub_components=None):
+    def __init__(self, component: Component, sub_components: Optional[List[str]] = None):
```

Comment:
Just a question on style here: Does the type hints syntax prefer whitespace around the equals sign?

Reply:
I think so: https://peps.python.org/pep-3107/#parameters.